tests/bluetooth/tester/src/btp/btp_vocs.h

```diff
@@ -0,0 +1,22 @@
+/* btp_vocs.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2022 Codecoup
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/sys/util.h>
+
+#define VOCS_READ_SUPPORTED_COMMANDS	0x01
+
+#define VOCS_UPDATE_LOC			0x02
+#define VOCS_UPDATE_DESC		0x03
+
+struct vocs_audio_desc {
+	uint8_t desc[8];
+} __packed;
+
+struct vocs_audio_loc {
+	uint8_t loc[8];
```

Comment:
Shouldn't this just be a `uint32_t`?

Reply:
if this is fixed size than ok,  if this is same as description than len+val[0]

Reply:
VOCS location is a 32-bit bitfield :) So it is a fixed size

Reply:
Done